packages/@lwc/module-resolver/src/__tests__/resolve.modules.spec.ts

```diff
@@ -263,7 +263,7 @@ describe('resolution override', () => {
 
         expect(() => resolveModule('test', dirname, opts)).toThrowErrorWithCode(
             LWC_CONFIG_ERROR_CODE,
-            `Invalid LWC configuration in "${dirname}". Invalid module record. Module record must be an object, instead got "foo/test".`
+            `Invalid LWC configuration in "inline_config". Invalid module record. Module record must be an object, instead got "foo/test".`
```

Comment:
This error message is not very intuitive. "inline_config" is not a standard term described in the RFC either(https://rfcs.lwc.dev/rfcs/lwc/0020-module-resolution). May be you want to use it as a phrase instead or have a completely different error message.